pkg/restore/restore.go

```diff
@@ -358,6 +358,7 @@ type context struct {
 	pvsToProvision             sets.String
 	pvRestorer                 PVRestorer
 	volumeSnapshots            []*volume.Snapshot
+	podVolumeBackups           []*api.PodVolumeBackup
```

Comment:
This field isn't currently being populated with data, but will be when #1691 merges.